lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -88,7 +89,7 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
                     converter: (store) => store.state.mySpaceState.viewState,
                     builder: (context, viewState) {
                       return viewState.fold(
-                          (failure) => Container(),
+                          (failure) => SizedBox.shrink(),
```

Comment:
Replaced `Container()` by `SizedBox.shrink()` for empty areas

=> https://stackoverflow.com/questions/53455358/how-to-present-an-empty-view-in-flutter

Reply:
nice